frontend/src/app/account-activation/account-activation.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, OnInit } from '@angular/core';
+import { Observable } from 'rxjs/Observable';
+import { AuthService } from '../auth.service';
+import { Router, ActivatedRoute, Params } from '@angular/router';
+
+@Component({
+  selector: 'volontulo-account-activation',
+  templateUrl: './account-activation.component.html',
+})
+export class AccountActivationComponent implements OnInit {
+  activationSuccessful: boolean;
+  failureMessage: string;
+  constructor(private authService: AuthService, private activatedRoute: ActivatedRoute
+  ) {
+  }
+
+  activate(uuid: string): void {
+    this.authService.activateAccount(uuid)
+      .subscribe(
+        rsp => {
+          this.activationSuccessful = rsp.status === 201;
+      },
+        err =>{
```

Comment:
above You have `rsp => {` and here is `err =>{` - can we make it coherent (I also belive, that's one of reasons, why Travis failed)